site/config.yaml

```
@@ -54,6 +54,17 @@ languages:
     languageName: English
     weight: 1
     contentDir: content/english
+    error:
+      title: Gone with the wind
+      subtitle: "This page doesn't exist or has moved.<br>We apologize for the inconvenience."
+      image:
+        alt:
+        filename: "error-404"
+        natural_width: 407
+      text: "You can return to home, or try looking on our <a href=\"https://help.skribble.com/english\">Help Center</a> if you need a hand. Perhaps you like 404 page. If so, you are welcome to stay for as long as you like :)"
```

```suggestion
      text: "You can return <a href=\"https://www.skribble.com\">home</a>, or try looking on our <a href=\"https://help.skribble.com/english\">Help Center</a> if you need a hand. If you really like our 404 page you are welcome to stay for as long as you like :)"
```
can we use 😉 or some other actual emoji instead of the :) Or are there too many browsers that don't support them and would lead to a broken character symbol?

---

Hm, meine Meinung bei solchen Sachen ist, dass die Leute kaputtes Internet gewohnt sind, wenn sie einen Browser verwenden, der das sonst auch nirgends kann.